core/utils/utils.py

```diff
@@ -276,3 +276,14 @@ def validate_bib_dir():
         return settings.BIB_STORAGE
     else:
         return None
+
+def cache_tag(response, tag_name):
```

Comment:
What do you think about naming this function `add_cache_tag`?

Reply:
👍